docs/source/guide/install.md

```diff
@@ -103,6 +111,39 @@ conda activate label-studio
 pip install label-studio
 ```
 
+## Troubleshoot installation
+
+You might see errors when installing Label Studio. Follow these steps to resolve them.
+
+### Run the latest version of Label Studio
+Many bugs might be fixed in patch releases or maintenance releases. Make sure you're running the latest version of Label Studio by upgrading your installation before you start Label Studio. 
+
+### Errors about missing packages
+
+If you see errors about missing packages, install those packages and try to install Label Studio again. Make sure that you run Label Studio in a clean Python environment, such as a virtual environment. 
+
+For example, you might need to run the following in an Anaconda environment:
```

Comment:
I'm not sure we need to provide the exact package installation commands here: prev paragraph is enough

Reply:
removed specific examples, agreed